weasyl/controllers/director.py

```diff
@@ -36,11 +36,12 @@ def directorcontrol_emailblacklist_post_(request):
                          selected_ids=map(int, form.remove_selection))
 
     # Add entry to blacklist
-    elif form.action == "add":
-        d.engine.execute(d.meta.tables["emailblacklist"].insert(), {
-            "domain_name": form.domain_name.lower(),
-            "reason": form.reason,
-            "added_by": request.userid,
-        })
+    elif form.action == "add" and form.domain_name:
+        if len(form.domain_name) <= 252:
```

Comment:
Not necessary (especially if you’re not going to produce a useful error); the database field already has a length limit.